jeta/archive/operations.py

```diff
@@ -16,56 +15,68 @@
 
 from astropy.time import Time
 
-import jeta.archive.file_defs as file_defs
 from jeta.archive.utils import get_env_variable
 
 ENG_ARCHIVE = get_env_variable('ENG_ARCHIVE')
 TELEMETRY_ARCHIVE = get_env_variable('TELEMETRY_ARCHIVE')
 ALL_KNOWN_MSID_METAFILE = get_env_variable('ALL_KNOWN_MSID_METAFILE')
-# JETA_LOGS = get_env_variable('JETA_LOGS')
 
+DEFAULT_CONFIG_PATH = '/srv/jeta/jeta/config/parameters.hdf5'
+
+
+def get_default_config(config_path=DEFAULT_CONFIG_PATH) -> str:
+    """ Get the default current config as json
+        Parameters
+        ----------
+        :return: serialized json str of the current system configuration
+    """
+    default_settings = defaultdict(list)
 
-# logger = pyyaks.logger.get_logger(
-#     filename=f'{JETA_LOGS}/jeta.operations.log',
-#     name='jeta_operations_logger',
-#     level='INFO',
-#     format="%(asctime)s %(message)s"
-# )
+    with h5py.File(config_path, 'r') as c:
+        for k in c.keys():
+            for a in c[k].attrs:
+                default_settings[k].append({a.upper(): str(c[k].attrs[a])})
+                os.environ[a.upper()] = str(c[k].attrs[a])
+    return json.dumps(default_settings)
 
 
-def set_config_parameter(subsystem: str=None, param: str=None, value: str=None) -> None:
+def set_config_parameter(
+    subsystem: str = None,
+    param: str = None,
+    value: str = None
+) -> None:
     """ Set the subsystem-level configuration in a persisted config file
         Parameters
         ----------
-        :param subsystem: the target subsystem of the configuration parameter (i.e. ingest, fetch, staging, archive)
+        :param subsystem: the target subsystem of the configuration parameter
         :param param: the name of the parameter value to set
         :param value: the value to set the parameter to
 
         :returns: config as str
     """
     if None in [subsystem, param, value]:
         raise ValueError('Subsystem, param, and value are required.')
-    with h5py.File(f'{TELEMETRY_ARCHIVE}/config/parameters.hdf5', 'a') as config:
-        if param.lower() in list(config[subsystem].attrs):
-            config[subsystem].attrs[param.lower()] = value
+    with h5py.File(f'{TELEMETRY_ARCHIVE}/config/parameters.hdf5', 'a') as c:
+        if param.lower() in list(c[subsystem].attrs):
+            c[subsystem].attrs[param.lower()] = value
         else:
             raise ValueError(f'{param} is not a valid system parameter.')
```

Comment:
Does the set_config_parameter() function only set existing parameters, and not create new ones?  Seems like the intent from the _update_current_config() function in docker/jeta/install_config.py is to use this function to create new parameters